modules/services/wlsunset.nix

```diff
@@ -1,17 +1,19 @@
-{ config, lib, pkgs, ... }:
-
-with lib;
-
-let cfg = config.services.wlsunset;
-
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.services.wlsunset;
 in {
-  meta.maintainers = [ hm.maintainers.matrss ];
+  meta.maintainers = [hm.maintainers.matrss];
```

Comment:
The formatting is off. The way it was previously should satisfy the code style of home-manager: https://nix-community.github.io/home-manager/index.xhtml#sec-code-style.

Reply:
I always use the Alejandra formatter so I assumed that you could use any formatter as long as it nix could evaluate it, I will change that